### ISSUE TITLE
Why can't the renamed "triton_python_backend_utils.py" and other modules be imported in models directory?

### ISSUE THREAD

**Issue by Fizmath** [triton-inference-server/server#2268]
Hello

this worked fine from your example   :  

![11](https://user-images.githubusercontent.com/40030502/99665250-6a233100-2a7a-11eb-85ad-cdfc84efe06f.png)

but when i renamed the triton_python_backend_utils.py  file to :  

![11](https://user-images.githubusercontent.com/40030502/99665463-b2daea00-2a7a-11eb-9ce1-c3d68fdd2999.png)

in ``model.py``
```import utility as pb_utils ```

got an error :  

``` ModuleNotFoundError: No module named 'utility' ```

this also happens when i want to import ``my``  own  modules , for example here 

![11](https://user-images.githubusercontent.com/40030502/99666150-a4410280-2a7b-11eb-9ac7-9fd3878d1771.png)

i get

``` ModuleNotFoundError: No module named 'process_1'  ```

I note that i just copied those files directly into docker volume and left python_backend_main folder out of it  :  

``` docker run --gpus=all --shm-size=1g --ulimit memlock=-1 -p 8000:8000 -p 8001:8001 -p 8002:8002 -v $PWD/models:/models --ulimit stack=67108864 -ti tritonserver:20.10-py3 ```

``` # tritonserver --model-repository /models ```

where i used dockerfile : 
```python
FROM nvcr.io/nvidia/tritonserver:20.10-py3

RUN pip3 install  --upgrade  --no-cache-dir  \
    torchvision

```
So where else should i put those new modules ?  

Another issue , what should i do  if my ``pytorch  script`` additionally loads pre-trained  ``*.pt`` file ?  here is the case :   

![11](https://user-images.githubusercontent.com/40030502/99669282-deac9e80-2a7f-11eb-8a18-91d8e1cc9248.png)

**Comment by Tabrizian** [comment]
@Fizmath Thanks for reporting this. I can confirm that there is an issue with importing modules in `model.py`. I will let you know about the fix when it is available. Regarding your second question, you can load the pytorch script in the `initialize` function:

```python
class TritonPythonModel:

    def initialize(self, args):
        model = TheModelClass(*args, **kwargs)
        model.load_state_dict(torch.load(PATH))
```

**Comment by Fizmath** [comment]
The example runs with impunity by skipping the step 4  :  

 ``` $ cp src/resources/triton_python_backend_utils.py  models/add_sub/1/ ``` 

I tracked down the problem :  

https://github.com/triton-inference-server/python_backend/blob/main/CMakeLists.txt#L232

So , shall we   ``Cmake``   any desired python  modules and  `` *.pt ``   files before importing in ``model.py``    ?

**Comment by Tabrizian** [comment]
This should be fixed in the current version of the Python backend. No, you shouldn't "Cmake" python modules. It should work by default now.

**Comment by gilevir** [comment]
Hi @Tabrizian ,

In which version this bug has been fixed ?
I am running triton version 2.5.0 from container 20.11 and i am still struggling importing custom python modules from `model.py`
As I am deploying Triton on GKE I cannot upgrade Triton to the very latest version.

**Comment by Tabrizian** [comment]
The fix is not available in the 20.11 release. It has been fixed in the 20.12 release.
